warehouse/integrations/snowflake/snowflake_test.go

```diff
@@ -965,56 +965,98 @@ func TestIntegration(t *testing.T) {
 
 func TestSnowflake_ShouldAppend(t *testing.T) {
```

Comment:
Perhaps we could add a test case with the users table too to make clear what's the desired behaviour there.

Reply:
I have added a tests case with tracks table, how does the users table differ?